source/_docs/international-regions.md

```diff
@@ -53,7 +61,7 @@ With this set of EU-specific resources, now you can run WordPress or Drupal site
 
   See `terminus site:create --help` for more information on the options and values used in this command.
 
-## Migrate an Existing Site to the EU Region
+## Migrate an Existing Site to an International Region
```

Comment:
Migrate an Existing Site Between Regions
or 
Migrate an Existing Site to a New Region